### PR TITLE
Fix password preview font, add some documentation

### LINKED ISSUE #3360: Highlight keepassxc-cli on the website and documentation
On twitter I saw that that the CLI was mentioned. 

Searching the repo found, of course, some entries https://github.com/keepassxreboot/keepassxc/search?q=cli

The manpage mentions the name `keepassxc-cli`. Ok, I can work from there. But it's really a bit hidden. 

Mention the CLI on the website, in the documentation and in the wiki. It is really a highlight to have it for scripting without having to install yet another application.

### LINKED ISSUE #3376: Use monospace font for password in preview window
## Summary
[NOTE]: # ( Provide a brief overview of what the new feature is all about )
When selecting one password entry the stored informations are displayed in the preview window. With the "eye" button you can see the stored password. It isn't used a monospace font, so you cannot distinguish between 0 and O and so on.

## Desired Behavior
[NOTE]: # ( Tell us how the new feature should work, be specific )
Please use a monospace font, as in the editing dialog window.

## Possible Solution
[NOTE]: # ( Not required, but suggest ideas on how to implement the addition or change )
Use monospace font for the password in the preview window as well, as in the edit dialogue box.

## Context
[NOTE]: # ( Why does this feature matter to you? What unique circumstances do you have? )
Example: Monospace font is used in editing dialog:
screenshot_keepassXC2

But not in preview:
screenshot_keepassXC3

### LINKED ISSUE #3378: Add keyboard shortcuts to documentation
I cannot find a documentation of the shortcuts working in keepassxc. Is there a documentation of the available shortcuts? The shortcuts I use, i found by trial and error, this is maybe not the best user experience.

### PR DESCRIPTION
**Pull request by SohamG**
[TIP]:  # ( Provide a general summary of your changes in the title above ^^ )

## Type of change
[NOTE]: # ( Please remove all lines which don't apply. )
- ✅ Bug fix (non-breaking change which fixes an issue)
- ✅ New feature (non-breaking change which adds functionality)
- ✅ Documentation (non-code change)

## Description and Context
- Fix #3376 
Use Font::fixedFont() for password preview

- #3378 (temporarily)
Added a menu entry in the Help menu which opens [the KEYBINDS document](https://github.com/keepassxreboot/keepassxc/blob/develop/docs/KEYBINDS.md)

- #3360 partially
Added a mention of the CLI to the README.

## Testing strategy
Manual

## Checklist:
[NOTE]: # ( Please go over all the following points. )
[NOTE]: # ( Again, remove any lines which don't apply. )
[NOTE]: # ( Pull Requests that don't fulfill all [REQUIRED] requisites are likely )
[NOTE]: # ( to be sent back to you for correction or will be rejected.  )
- ✅ I have read the **CONTRIBUTING** document. **[REQUIRED]**
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ I have compiled and verified my code with `-DWITH_ASAN=ON`. **[REQUIRED]**
- ✅ My change requires a change to the documentation, and I have updated it accordingly.

### CODE REVIEW

**Comment by SohamG** [comment]
@droidmonkey Please can you review

**Comment by droidmonkey** [comment]
Yes been fairly busy lately. I'm going to clear PR's later this werk

**Comment by SohamG** [comment]
Sure no issues

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ so please check out your distribution's package list to see if KeePassXC is avai
 - Automatic reload when the database changed on disk
 - Browser integration with KeePassXC-Browser using [native messaging](https://developer.chrome.com/extensions/nativeMessaging) for [Mozilla Firefox](https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/) and [Google Chrome, Chromium, Vivaldi, or Brave](https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk)
 - Synchronize passwords using KeeShare. See [Using Sharing](./docs/QUICKSTART.md#using-sharing) for more details.
+- A [Command Line Interface (keepassxc-cli)](./src/cli/keepassxc-cli.1)
```
Comment (louib) [inline README.md]: @SohamG It's actually already mentioned just above (line 25)! Maybe replace that line instead?

Comment (SohamG) [reply README.md]: I didn't spot that, fixed now!

File: src/gui/MainWindow.cpp [hunk 2]
```diff
@@ -814,6 +815,12 @@ void MainWindow::openOnlineHelp()
 {
     customOpenUrl("https://keepassxc.org/docs/");
 }
+
+void MainWindow::openKbdShortcuts()
```
Comment (droidmonkey) [inline src/gui/MainWindow.cpp]: I prefer a full name: `openKeyboardShortcuts()`

File: src/gui/MainWindow.ui [hunk 3]
```diff
@@ -234,6 +234,7 @@
     <addaction name="separator"/>
     <addaction name="actionGettingStarted"/>
     <addaction name="actionUserGuide"/>
+    <addaction name="actionKbdShortcuts"/>
```
Comment (droidmonkey) [inline src/gui/MainWindow.ui]: Full name preferred

**Comment by SohamG** [comment]
Noted. I'll update as soon as possible.

File: src/gui/MainWindow.ui (hunk 3 above)
Comment (SohamG) [reply src/gui/MainWindow.ui]: Done!

**Comment by SohamG** [comment]
@droidmonkey Ive made the changes requested

**Pull request merged**